### PR TITLE
OPSEXP-2109: Release for acs7.4

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-2109

### CODE REVIEW

File: docker-compose/7.1.N-docker-compose.yml [hunk 1]
```diff
@@ -121,7 +121,7 @@ services:
     ports:
       - "5432:5432"
   solr6:
-    image: alfresco/alfresco-search-services:2.0.2.2
+    image: quay.io/alfresco/search-services:2.0.2
```
Comment (gionn) [inline docker-compose/7.1.N-docker-compose.yml:124]:
```suggestion
    image: quay.io/alfresco/search-services:2.0.2.2
```

File: docker-compose/7.1.N-docker-compose.yml [hunk 2]
```diff
@@ -135,17 +135,17 @@ services:
...
-      - "5672:5672" # AMQP
-      - "61616:61616" # OpenWire
-      - "61613:61613" # STOMP
+      - "8161:8161"  # Web Console
+      - "5672:5672"  # AMQP
+      - "61616:61616"  # OpenWire
+      - "61613:61613"  # STOMP
   digital-workspace:
-    image: quay.io/alfresco/alfresco-digital-workspace:2.6.2
+    image: quay.io/alfresco/alfresco-digital-workspace:2.6.1
```
Comment (gionn) [inline docker-compose/7.1.N-docker-compose.yml:148]:
```suggestion
    image: quay.io/alfresco/alfresco-digital-workspace:2.6.2
```

File: docker-compose/7.2.N-docker-compose.yml [hunk 3]
```diff
@@ -177,7 +177,7 @@ services:
       - share
       - control-center
   sync-service:
-    image: quay.io/alfresco/service-sync:3.8.0
+    image: quay.io/alfresco/service-sync:3.7.2
```
Comment (gionn) [inline docker-compose/7.2.N-docker-compose.yml:180]:
```suggestion
    image: quay.io/alfresco/service-sync:3.8.0
```

File: helm/alfresco-content-services/7.3.N_values.yaml [hunk 4]
```diff
@@ -23,43 +22,43 @@ transformmisc:
...
-    tag: 1.4.5
+    tag: 1.5.0
 filestore:
   image:
-    tag: 2.0.0
+    tag: 2.0.2
 share:
   image:
-    tag: 7.3.1
+    tag: 7.3.0.5
```
Comment (gionn) [inline helm/alfresco-content-services/7.3.N_values.yaml:31]:
```suggestion
    tag: 7.3.1
```

File: helm/alfresco-content-services/7.2.N_values.yaml [hunk 5]
```diff
@@ -1,61 +1,60 @@
...
 alfresco-search:
   searchServicesImage:
-    tag: 2.0.4
+    tag: 2.0.7
 alfresco-search-enterprise:
   liveIndexing:
     mediation:
       image:
-        tag: 3.1.1
+        tag: 3.3.0
```
Comment (gionn) [inline helm/alfresco-content-services/7.2.N_values.yaml:39]:
```suggestion
        tag: 3.1.1
```

File: helm/alfresco-content-services/7.2.N_values.yaml [hunk 6]
```diff
@@ -1,61 +1,60 @@
...
     content:
       image:
-        tag: 3.1.1
+        tag: 3.3.0
```
Comment (gionn) [inline helm/alfresco-content-services/7.2.N_values.yaml:42]:
```suggestion
        tag: 3.1.1
```

File: helm/alfresco-content-services/7.2.N_values.yaml [hunk 7]
```diff
@@ -1,61 +1,60 @@
...
     metadata:
       image:
-        tag: 3.1.1
+        tag: 3.3.0
```
Comment (gionn) [inline helm/alfresco-content-services/7.2.N_values.yaml:45]:
```suggestion
        tag: 3.1.1
```

File: helm/alfresco-content-services/7.2.N_values.yaml [hunk 8]
```diff
@@ -1,61 +1,60 @@
...
     path:
       image:
-        tag: 3.1.1
+        tag: 3.3.0
```
Comment (gionn) [inline helm/alfresco-content-services/7.2.N_values.yaml:48]:
```suggestion
        tag: 3.1.1
```

File: helm/alfresco-content-services/7.2.N_values.yaml [hunk 9]
```diff
@@ -1,61 +1,60 @@
...
   reindexing:
     image:
-      tag: 3.1.1
+      tag: 3.3.0
```
Comment (gionn) [inline helm/alfresco-content-services/7.2.N_values.yaml:51]:
```suggestion
      tag: 3.1.1
```

File: helm/alfresco-content-services/7.1.N_values.yaml [hunk 10]
```diff
@@ -1,64 +1,63 @@
...
 alfresco-search:
   searchServicesImage:
-    tag: 2.0.2.2
+    tag: 2.0.2
 alfresco-search-enterprise:
   liveIndexing:
     mediation:
       image:
-        tag: 3.1.1
+        tag: 3.3.0
```
Comment (gionn) [inline helm/alfresco-content-services/7.1.N_values.yaml:39]:
```suggestion
        tag: 3.1.1
```

File: helm/alfresco-content-services/7.1.N_values.yaml [hunk 11]
```diff
@@ -1,64 +1,63 @@
...
     content:
       image:
-        tag: 3.1.1
+        tag: 3.3.0
```
Comment (gionn) [inline helm/alfresco-content-services/7.1.N_values.yaml:42]:
```suggestion
        tag: 3.1.1
```

File: helm/alfresco-content-services/7.1.N_values.yaml [hunk 12]
```diff
@@ -1,64 +1,63 @@
...
     metadata:
       image:
-        tag: 3.1.1
+        tag: 3.3.0
```
Comment (gionn) [inline helm/alfresco-content-services/7.1.N_values.yaml:45]:
```suggestion
        tag: 3.1.1
```

File: helm/alfresco-content-services/7.1.N_values.yaml [hunk 13]
```diff
@@ -1,64 +1,63 @@
...
     path:
       image:
-        tag: 3.1.1
+        tag: 3.3.0
   reindexing:
     image:
-      tag: 3.1.1
+      tag: 3.3.0
```
Comment (gionn) [inline helm/alfresco-content-services/7.1.N_values.yaml:51]:
```suggestion
      tag: 3.1.1
```

**Review by gionn — COMMENTED** [review]
to revert as not conform with supported platform

File: docker-compose/7.0.N-docker-compose.yml [hunk 14]
```diff
@@ -121,7 +121,7 @@ services:
     ports:
       - "5432:5432"
   solr6:
-    image: alfresco/alfresco-search-services:2.0.2.2
+    image: quay.io/alfresco/search-services:2.0.2
```
Comment (gionn) [inline docker-compose/7.0.N-docker-compose.yml:124]:
```suggestion
    image: quay.io/alfresco/search-services:2.0.2.2
```

File: docker-compose/7.3.N-docker-compose.yml [hunk 15]
```diff
@@ -123,7 +122,7 @@ services:
     ports:
       - "5432:5432"
   solr6:
-    image: alfresco/alfresco-search-services:2.0.5
+    image: quay.io/alfresco/search-services:2.0.7
```
Comment (gionn) [inline docker-compose/7.3.N-docker-compose.yml:125]:
```suggestion
    image: quay.io/alfresco/search-services:2.0.5
```

**Review by slohe1 — APPROVED** [review]
LGTM

**Review by gionn — APPROVED** [review]

**Pull request merged**